sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -75,10 +87,14 @@ class ProjectOverviewView extends VerticalLayout{
         viewModel.addPropertyChangeListener("selectedProject", {
             Optional<ProjectSummary> selectedProject = Optional.ofNullable(viewModel.selectedProject)
             selectedProject.ifPresent({
-                loadFailedQcSamples(it)
+                if(failingSamplesExist()){
+                    loadFailedQcSamples(it)
+                    failedQCSamplesView.setVisible(true)
```

Comment:
I think you do not need this line as the component `failedQCSamplesView` can be set up by calling `showWhenFailingSamplesExist`. This way the behaviour would be abstracted out of the closure that is executed if a selected project is present. This would make the separation of responsibility more clear.

Reply:
good point, this was a leftover from some testing, I removed them 👍🏼